### ISSUE TITLE
Multi-Calendar Sync Breaks When GCal event deleted but not Notion item

### ISSUE THREAD

**Issue by akarri2001** [akarri2001/Notion-and-Google-Calendar-2-Way-Sync#5]
If you delete the GCal event and not the Notion item, the calendar sync is inaccurate for other events. However, if you delete the Notion item and run the code again, then everything is fixed again.

**Comment by akarri2001** [comment]
I'm working on developing some kind of "deletion sync" to not have to worry about this problem in the future

**Comment by suged** [comment]
Thanks for the great work.
It means at this time if I delete a Notion item, it will not be deleted from gcal ? Is there a simple fix ?

**Comment by akarri2001** [comment]
Yes, that is correct; if the item is deleted from Notion and not from gCal, then the code will add it back into Notion. A fix would be adding a "delete" checkbox property to the Notion database and then having the code delete all of the items where it is checked on both gCal and Notion. 

I'm pretty busy right now with work so developing it will be on the horizon but I can't say exactly when.
